infra/src/main/java/com/redhat/service/bridge/infra/transformations/TransformationEvaluatorFactoryQute.java

```diff
@@ -0,0 +1,27 @@
+package com.redhat.service.bridge.infra.transformations;
+
+import javax.enterprise.context.ApplicationScoped;
+
+import io.quarkus.qute.Engine;
+import io.quarkus.qute.TemplateException;
+
+@ApplicationScoped
+public class TransformationEvaluatorFactoryQute implements TransformationEvaluatorFactory {
+
+    private static final Engine engine = Engine.builder().addDefaults().build();
+
+    @Override
+    public TransformationEvaluator build(String template) {
+        return new TransformationEvaluatorQute(engine, template);
+    }
+
+    @Override
+    public TransformationValidationResult validate(String template) {
+        try {
+            engine.parse(template);
+        } catch (TemplateException e) {
+            return new TransformationValidationResult(false, e.getMessage());
+        }
+        return new TransformationValidationResult(true, null);
```

Comment:
Maybe we could have 2 constructors to avoid need passing null arguments
```
public TransformationValidationResult(boolean isValid)

public TransformationValidationResult(boolean isValid, String reason)
```